FinniversKit/Sources/Components/HTMLText/HTMLTextFinnStyle.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+import SwiftUI
+import UIKit
+
+extension HTMLStringSwiftUIStyleTranslator {
+    static func finnStyle(
+        font: Font?,
+        foregroundColor: Color?
+    ) -> HTMLStringSwiftUIStyleTranslator {
+        return .init(defaultStyle: .init(
+            font: font,
+            foregroundColor: foregroundColor
+        )) { elementName, attributes in
+            var style = HTMLStringSwiftUIStyleTranslator.Style()
+            switch elementName.lowercased() {
+            case "b", "strong":
```

Comment:
Sure, but would it help since any styling will be heavily customized? There won't necessarily be a single Finn styler either, and there has been talk of custom tags (but I think that was dropped). I struggled a bit with coming up with a flexible way of defining styling. Improvements are welcome.

Reply:
Yes, I see your points on this, but just seing so many string like "b", "i" etc feels wrong, so just having an enum in this file might look nicer 🤔 Not a requirement to change this though